HelperFunctions.ps1

```diff
@@ -1276,6 +1282,11 @@ function RunAlTool {
     if ($isLinux) {
         $alToolExe = Join-Path $path 'extension/bin/linux/altool'
         & /usr/bin/env sudo pwsh -command "& chmod +x $alToolExe"
+    } 
+    elseif ($isMacOS) {
+        $alToolExe = Join-Path $path 'extension/bin/darwin/altool'
+        Write-Host "Setting execute permissions on altool"
+        & chmod +x $alToolExe
```

Comment:
No sudo needed?

Reply:
Correct, worked without it since it's a user repository.